setup.py

```diff
@@ -47,6 +47,8 @@ def unique_flatten_dict(d):
 # https://github.com/facebookresearch/faiss/issues/1589 for faiss-cpu 1.6.1, #'setuptools==67.4.0' removed
 base_extras_heavy['ai'] = base_extras_heavy['umap-learn'] + ['scipy', 'dgl', 'torch<2', 'sentence-transformers', 'faiss-cpu', 'joblib']
 
+base_extras_heavy['cu_cat'] = base_extras_heavy['ai'] + ['cu_cat @ git+http://github.com/graphistry/cu-cat.git@v0.04.0']
```

Comment:
`graphistry/cu-cat` is 159 commits behind `dcolinmorgan:master`, does that need to land before we can install cu_cat in the dockerfiles?